src/main/resources/assets/js/application/interviewer/controllers/tdprInterviewerHomeController.js

```diff
@@ -181,18 +181,24 @@ define(['angular', 'application/interviewer/tdprInterviewerModule', 'application
                 if(failure.status === 406) {
                     $scope.temporaryContent = $scope.noteContent.description;
                 }
-                    Notification.warning({
-                        message: 'Something went wrong with sending your note.',
-                        delay: 2000});
+                Notification.warning({
+                    message: 'Something went wrong with sending your note.',
+                    delay: 2000});
             });
         }
 
         $scope.markSlots = function(slot) {
-            if($scope.relativeDayNumber < 0 && !$scope.isRecruiter){
-                return;
+            if(!$scope.isRecruiter) {
+                if(slot.type === AvailabilityEnum.full.id ||
```

Comment:
Complex conditional expressions obscure code readability. Consider extracting the content of this if into a function with a meaningful name instead.